content/foundations/color/overview.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: UI color system
+description:
+  Color is a fundamental part of Primer's visual language. In this guide you will learn about the principles,
+  patterns, and abstractions that make up Primer’s color system.
+---
+
+import {Box} from '@primer/react'
+
+## Overview
```

Comment:
**Suggestion V2:** 
I wonder if we can have some sort of quick links to help users navigate and find helpful pages regarding color. If I instinctively click **color** and am looking for the reference, I might not think to look under primitives. Probably a V2, I just wonder if we've considered a pattern like this in our docs before to help with navigation / IA.

Maybe something like: 
<img width="935" alt="3 cards UI" src="https://github.com/primer/design/assets/7265547/e4b42b62-a9d8-4ede-bd6d-273342ad1a6f">